<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddProduct</class>
 <widget class="QDialog" name="AddProduct">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>662</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Beheer Producten</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource.qrc">
    <normaloff>:/icons/icons/file_manager.png</normaloff>:/icons/icons/file_manager.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>481</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>41</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <widget class="QPushButton" name="btnNieuw">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../resource.qrc">
        <normaloff>:/icons/icons/add.png</normaloff>:/icons/icons/add.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="btnBewerk">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>0</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../resource.qrc">
        <normaloff>:/icons/icons/edit_button.png</normaloff>:/icons/icons/edit_button.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="btnOpslaan">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>0</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../resource.qrc">
        <normaloff>:/icons/icons/save_close.png</normaloff>:/icons/icons/save_close.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="btnStop">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>0</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../resource.qrc">
        <normaloff>:/icons/icons/stop2.png</normaloff>:/icons/icons/stop2.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLaadPicture">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>0</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../resource.qrc">
        <normaloff>:/icons/icons/picture_add.png</normaloff>:/icons/icons/picture_add.png</iconset>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>481</width>
       <height>106</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>106</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <widget class="my_lbFoto" name="lbFoto">
      <property name="geometry">
       <rect>
        <x>375</x>
        <y>5</y>
        <width>101</width>
        <height>96</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(231, 231, 231);</string>
      </property>
      <property name="text">
       <string>img</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lbCode">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>10</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Code:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbBenaming">
      <property name="geometry">
       <rect>
        <x>95</x>
        <y>10</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Benaming:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbOmschrijving">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>40</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Omschrijving:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="tbBenaming">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>5</y>
        <width>196</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>tbBenaming</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="tbOmschrijving">
      <property name="geometry">
       <rect>
        <x>95</x>
        <y>35</y>
        <width>271</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>tbOmschrijving</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbxCode">
      <property name="geometry">
       <rect>
        <x>45</x>
        <y>5</y>
        <width>41</width>
        <height>27</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="maxCount">
       <number>36</number>
      </property>
      <item>
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>A</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>B</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>C</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>D</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>E</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>F</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>G</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>H</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>I</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>J</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>K</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>L</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>M</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>N</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>O</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>P</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Q</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>R</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>S</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>T</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>U</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>V</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>W</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>X</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Y</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Z</string>
       </property>
      </item>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tvProducten">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>42</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>42</number>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>my_lbFoto</class>
   <extends>QLabel</extends>
   <header>my_lbfoto.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
